src/sst/core/configGraph.cc

```diff
@@ -434,10 +515,8 @@ ConfigGraph::postCreationCleanup()
     for ( ConfigLink &link : getLinkMap() ) {
         link.updateLatencies(timeLord);
     }
-
 }
 
```

Comment:
We try to limit any whitespace changes in each commit.

Reply:
👍 14178e3d8466f961c368327c6f18eaa558ae7774